### PR TITLE
Describe how we should handle the documentation part of the project

### PR DESCRIPTION
**Pull request by adiroiban**
# Scope

This tries to add an initial few ideas and notes about how we should document the project.

from technical docstrings, to end user documentation to marketing materials

if we would have to start this project from zero and we would all have our project related info erased, this document should include all common knowledge and best practices that we have found while working on the project.
# Changes

For now, there is just a structure
# How to try and test the changes

reviewers: @alibotean @brunogola @atykhonov @lgheorghiu @dumol 

I have just pushed an initial page structure.. if know know what we can put here, please push your changes :)

Thanks!

### CODE REVIEW

**Comment by adiroiban** [comment]
In the following days I will try to add more info here... but meanwhile if you know what should be here, please contribute.

Thanks!

File: content/documentation.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+talk about different documentation formats and different audiance
+
+
+From /generic.html talk about newliens and how we manage diffs .. move that
+part here.
+
+For narrative documentation use semantic newlines.
+Make lines short, and break lines at natural places,
+such as after commas and semicolons, rather than after the Nth column.
```
Comment (atykhonov) [inline content/documentation.rst]: @adiroiban I wonder why? Is documentation more readable with the semantic newlines?

Comment (adiroiban) [inline content/documentation.rst]: with semantic newlines the diff are much easier to read

**Comment by hcs0** [comment]
Just noticed this PR which is similar to what I have in mind for https://github.com/chevah/styleguide/issues/44 :)
I think it's better to start a-new in case there has been some changes from the styleguide since then (plus there is a Trac ID associated).

**Comment by adiroiban** [comment]
@hcs0 Thanks. yes. is ok to have a new one. Please consider including the tentative section from this PR so that this PR can be closed as duplicate.

**Comment by hcs0** [comment]
Closing due to duplicate of https://github.com/chevah/styleguide/tree/3858-documentation

**Pull request closed without merging**